drivers/pinmux/stm32/pinmux_stm32f0.h

```diff
@@ -164,11 +164,21 @@
 #define STM32F0_PINMUX_FUNC_PB9_I2C1_SDA \
 	(STM32_PINMUX_ALT_FUNC_1 | STM32_OPENDRAIN_PULLUP)
 
-#define STM32F0_PINMUX_FUNC_PF0_I2C1_SCL \
+#define STM32F0_PINMUX_FUNC_PB10_I2C1_SCL \
 	(STM32_PINMUX_ALT_FUNC_1 | STM32_OPENDRAIN_PULLUP)
-#define STM32F0_PINMUX_FUNC_PF1_I2C1_SDA \
+#define STM32F0_PINMUX_FUNC_PB11_I2C1_SDA \
 	(STM32_PINMUX_ALT_FUNC_1 | STM32_OPENDRAIN_PULLUP)
 
+#define STM32F0_PINMUX_FUNC_PF1_I2C1_SCL \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_OPENDRAIN_PULLUP)
+#define STM32F0_PINMUX_FUNC_PF0_I2C1_SDA \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_OPENDRAIN_PULLUP)
```

Comment:
As far as I can see, the defines are normally ordered by pin number (PF0 before PF1).